ooc-sdk-lite.use

```diff
@@ -1,4 +1,17 @@
 Name: sdk
 Description: Lightweight ooc standard library replacement.
 SourcePath: source/sdk
-Imports: lang/Abstractions, lang/Buffer, lang/BufferIterator, lang/Character, lang/Exception, lang/Format, lang/IO, lang/Iterators, lang/Memory, lang/Numbers, lang/String, lang/stdlib, lang/types, lang/VarArgs
+Imports: lang/Abstractions
+Imports: lang/Buffer
+Imports: lang/BufferIterator
+Imports: lang/Character
+Imports: lang/Exception
+Imports: lang/Format
+Imports: lang/IO
+Imports: lang/Iterators
+Imports: lang/Memory
+Imports: lang/Numbers
+Imports: lang/String
+Imports: lang/stdlib
+Imports: lang/types
+Imports: lang/VarArgs
```

Comment:
Hmm. If all of these classes are in `lang`, can't we change the `SourcePath` to `source/sdk/lang`?